components/database/mongodb-34/Makefile

```diff
@@ -19,6 +19,7 @@ COMPONENT_NAME=		mongodb
 COMPONENT_MAJOR_VERSION=3.4
 COMPONENT_MINOR_VERSION=19
 COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
+COMPONENT_REVISION=     1
 COMPONENT_SUMMARY=	MongoDB distributed document-oriented database
```

Comment:
Something seems to be missing here. Is COMPONENT_REVISION the only thing to change here? At least I would expect to make use of common.mk.

Reply:
Forgot to push the patch. I will modify the components to use common.mk only when the PR is to the point that we can migrate to gcc 7.5, my priority is to fix the broken components first since I do not have much spare time lately :S

Reply:
Welcome to the club. It will get worse when you have children :-)
BTW: Most broken builds aren't broken because of gcc-7.5. So I guess you can already merge fixes.

Reply:
Yes you are right :) The list of broken components should be up-to-date, I hope I can fix the lisp stuff next week...

Reply:
Unfortunately recent PRs broke more components... Could we decide to work on the gcc migration over one week and get past this task for good?

Reply:
Sorry, about that. I stop integrating new stuff until this one is done (except for https://github.com/OpenIndiana/oi-userland/pull/5470, which really needs to be done now).

Reply:
What about removing (obsolete) some packages?
icedtea-web is a candidate to remove.
I want to remove old node and node6 packages.
Do we need all the golang and gcc versions?

Reply:
> Sorry, about that. I stop integrating new stuff until this one is done (except for #5470, which really needs to be done now).

I is fine, I had missed your gnutls PR :) lonely packages are not a problem, only PRs with dependencies so that manually intervention is involved. Even gstreamer is not an issue since it is small.
There is not much work involved to move to gcc 7.5 so this could be something that we prioritize to get over it.

Reply:
> What about removing (obsolete) some packages?

That's always a good idea :).

> icedtea-web is a candidate to remove.

I am not opposing but why? It's not working in Firefox, useless, or...?

> I want to remove old node and node6 packages.

Sure, go ahead.

> Do we need all the golang and gcc versions?

Can't speak for GCC :), but for Go we can drop all obsolete ones unless they are used for build of a supported one (unless we also remove the supported one) - right @Toasterson?

Reply:
> There is not much work involved to move to gcc 7.5 so this could be something that we prioritize to get over it.

Cool. How do we test those GCC 7.5 rebuilt packages? For such a change I'd like to have install images to test that things did not explode. Can you prepare them? Or perhaps I can build them if there's IPS repo I can use?

Reply:
icedtea-web is fairly outdated and our version uses Java 7 (the latest uses Java 8).
Nobody seems to care and who uses Java applets nowadays?
Plus, it doesn't build for me at the moment...

Reply:
@AndWac Thanks for the explanation, drop it then.

Reply:
This is not only about java applets, but also about javaws utility. And this one is really useful and working even now.